nixos/modules/flyingcircus/roles/webproxy.nix

```diff
@@ -40,12 +52,30 @@ in
          (fclib.listenAddressesQuotedV6 config "lo"));
     services.varnish.config = varnishCfg;
     services.varnish.stateDir = "/var/spool/varnish/${config.networking.hostName}";
-    # XXX: needs to be migrated to varnish service
-    systemd.services.varnish.after = [ "network.target" ];
+    systemd.services.varnish = {
+      # XXX: needs to be migrated to varnish service
```

Comment:
I don't understand the comment...

Reply:
It was there before. It means that this actually should go upstream